frontend/src/components/HeaderDropdown/HeaderDropdown.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import {
+  Dropdown, DropdownToggle, DropdownMenu, DropdownItem,
+} from 'reactstrap';
+import './HeaderDropdown.css';
+import profilePic from '../../assets/profilePic.PNG';
+
+
+class HeaderDropdown extends Component {
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      dropdownOpen: false,
+    };
+  }
+
+  toggle(state) {
```

Comment:
NitPick: Let's make this method more descriptive. Maybe something like `toggleDropdown`.